pr2eus_tutorials/launch/pr2_tabletop.launch

```diff
@@ -5,11 +5,25 @@
   <arg name="manager" default="pr2_tabletop_object_detector_nodelet_manager"/>
   <arg name="machine" default="c2" />
```

Comment:
If you use this launch file from remote, machine is localhost?

Reply:
Fix with [commit 791eb8682f830106bf868e7405cd555da150ab2a](https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/459/commits/791eb8682f830106bf868e7405cd555da150ab2a)